pkgs/applications/networking/instant-messengers/telegram-cli/default.nix

```diff
@@ -11,14 +11,17 @@ stdenv.mkDerivation rec {
     rev = "2052f4b381337d75e783facdbfad56b04dec1a9c";
   };
 
-  buildInputs = [ libconfig lua5_2 openssl readline zlib libevent pkgconfig python jansson ];
+  buildInputs = [
+    libconfig libevent openssl readline zlib
+    lua5_2 python pkgconfig jansson
+  ];
   installPhase = ''
     mkdir -p $out/bin
     cp ./bin/telegram-cli $out/bin/telegram-wo-key
     cp ./tg-server.pub $out/
     cat > $out/bin/telegram-cli <<EOF
     #!${bash}/bin/sh
-    $out/bin/telegram-wo-key -k $out/tg-server.pub
+    $out/bin/telegram-wo-key -k $out/tg-server.pub "\$@"
```

Comment:
I use telegram-cli for this [program](https://gist.github.com/rnhmjoj/5f42ce3f31f8be88e921#file-telegram-hs). It stopped working when I moved to nixOS for this reason but it runs fine with this patch now.